streams/src/main/java/org/apache/kafka/streams/processor/ThreadMetadata.java

```diff
@@ -73,12 +123,16 @@ public boolean equals(final Object o) {
         return Objects.equals(threadName, that.threadName) &&
                Objects.equals(threadState, that.threadState) &&
                Objects.equals(activeTasks, that.activeTasks) &&
-               Objects.equals(standbyTasks, that.standbyTasks);
+               Objects.equals(standbyTasks, that.standbyTasks) &&
+               mainConsumerClientId.equals(that.mainConsumerClientId) &&
+               restoreConsumerClientId.equals(that.restoreConsumerClientId) &&
+               Objects.equals(producerClientIds, that.producerClientIds) &&
+               adminClientId.equals(that.adminClientId);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(threadName, threadState, activeTasks, standbyTasks);
+        return Objects.hash(threadName, threadState, activeTasks, standbyTasks, mainConsumerClientId, restoreConsumerClientId, producerClientIds, adminClientId);
```

Comment:
Nit: Can we break this long line into one parameter per line?

Reply:
ack.